sdk/eventhub/event-hubs/package.json

```diff
@@ -58,7 +58,7 @@
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace dist-esm/test/*.spec.js dist-esm/test/**/*.spec.js",
+    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace dist-esm/test/public/*.spec.js dist-esm/test/internal/*.spec.js dist-esm/test/public/**/*.spec.js dist-esm/test/internal/**/*.spec.js",
```

Comment:
@willmtemple  - tell me that someday we'll make devtool compose this line (or just invoke devtool directly).

Reply:
For metrics-advisor we are using devtool

Reply:
@richardpark-msft What about something like `dev-tool run-default integration-test:node` that would be able to run a standard default configuration?

Reply:
Ultimately my hope is that this can be supported in Rush, since it seems hacky to do such a thing in dev-tool.